src/commonTest/kotlin/com.adamratzman/spotify/utilities/UrisTests.kt

```diff
@@ -349,40 +362,7 @@ class UrisTests : Spek({
             }
         }
     }
-    describe("Uri serialization test") {
-        val json = Json(JsonConfiguration.Stable)
-        it("create UserUri from json by using SpotifyUri.serializer()") {
-            val spotifyUri: SpotifyUri =
-                json.parse(SpotifyUri.serializer(), "\"spotify:user:7r7uq6qxa4ymx3wnjd9mm6i83\"")
-            assertEquals(
-                UserUri::class,
-                spotifyUri::class
-            )
-            assertEquals(
-                "spotify:user:7r7uq6qxa4ymx3wnjd9mm6i83",
-                spotifyUri.uri
-            )
-        }
-        it("create UserUri from json by using UserUri.serializer()") {
-            val userUri = json.parse(UserUri.serializer(), "\"spotify:user:7r7uq6qxa4ymx3wnjd9mm6i83\"")
-            assertEquals(
-                "spotify:user:7r7uq6qxa4ymx3wnjd9mm6i83",
-                userUri.uri
-            )
-        }
-        it("try creating UserUri from json with id by using SpotifyUri.serializer()") {
-            assertFailsWith<SpotifyUriException> {
-                json.parse(SpotifyUri.serializer(), "\"7r7uq6qxa4ymx3wnjd9mm6i83\"")
-            }
-        }
-        it("create UserUri from json with id by using UserUri.serializer()") {
-            val userUri = json.parse(UserUri.serializer(), "\"7r7uq6qxa4ymx3wnjd9mm6i83\"")
-            assertEquals(
-                "spotify:user:7r7uq6qxa4ymx3wnjd9mm6i83",
-                userUri.uri
-            )
-        }
-    }
```

Comment:
Why did you removed it? That should still work if you want the library to parse any Uris.